crates/app/tests/mod.rs

```diff
@@ -89,12 +95,117 @@ async fn app() {
     )
     .await;
 
+    println!("Put testing file in foo tag");
+
     let (test_resp, test_type) =
         tree::get_path(&cl, &addr, &foo, &tag, &"/".parse().unwrap()).await;
 
     assert_eq!(test_type, mime::TEXT_PLAIN);
     assert_eq!(&test_resp[..], b"testing");
 
+    let random_bytes = [
+        0x8Bu8, 0x3Eu8, 0x7Eu8, 0x57u8, 0x05u8, 0x73u8, 0xEEu8, 0xDFu8, 0xE6u8, 0xD2u8, 0xCBu8,
+        0xAAu8, 0xD0u8, 0x19u8, 0x3Fu8, 0xABu8, 0xDDu8, 0x38u8, 0x34u8, 0x11u8, 0x7Du8, 0x51u8,
+        0xC5u8,
+    ];
+
+    let random_bytes_digest = Algorithms::default()
+        .read(random_bytes.as_ref())
+        .await
+        .unwrap();
+
+    let random_bytes_sha256 = random_bytes_digest
+        .get(&Algorithm::Sha256)
+        .unwrap()
+        .to_vec();
+
+    let tag_2 = "0.2.0".parse().unwrap();
+    println!("Creating signed tag.");
+    tag::create(
+        &cl,
+        &addr,
+        &foo,
+        &tag_2,
+        Jws::TYPE,
+        TagEntry::Signed(Jws::Flattened(Flattened {
+            payload: Some(Bytes::from(random_bytes_sha256)),
```

Comment:
Why not just store a constant `"test"`? What's the benefit of using a hash here?